src/chocolatey.console/targets/Microsoft.Application.targets

```diff
@@ -22,7 +22,7 @@ Copyright (C) 2005 Microsoft Corporation. All rights reserved.
     <ExeProjectOutputDirInsideProject Condition="'$(ExeProjectOutputDirInsideProject)' == ''">$(ExeProjectOutputDirInsideProjectDefault)</ExeProjectOutputDirInsideProject>
     <ExeProjectOutputDir>$(MSBuildProjectDirectory)</ExeProjectOutputDir>
     <ExeProjectOutputDir Condition="!$(ExeProjectOutputDirInsideProject)">$(OutDir)_PublishedApplications\$(MSBuildProjectName)</ExeProjectOutputDir>
-    <DisableLinkInCopyExeApplicaton Condition="'$(DisableLinkInCopyExeApplicaton)'==''">False</DisableLinkInCopyExeApplicaton>
+    <DisableLinkInCopyExeApplication Condition="'$(DisableLinkInCopyExeApplication)'==''">False</DisableLinkInCopyExeApplication>

```

Comment:
There aren't many instances of this around, but this appears to be self contained, so I think it's ok to change this.